lib/check_for_oplog.js

```diff
@@ -197,23 +145,23 @@ OplogCheck.limitButNoSort = function (cursorDescription) {
   return true;
 };
 
-OplogCheck.olderVersion = function (cursorDescription, driver) {
-  if (driver && !driver.constructor.cursorSupported) {
+OplogCheck.thirdParty = function (cursorDescription, observerDriver) {
+  if (Tracker.active && observerDriver.constructor.name !== 'OplogObserveDriver') {
     return {
-      code: 'OLDER_VERSION',
-      reason: 'Your Meteor version does not have oplog support.',
-      solution: 'Upgrade your app to Meteor version 0.7.2 or later.'
+      code: 'TRACKER_ACTIVE',
+      reason: 'Observe driver detected inside an active tracker, you might be using a third party library (e.g "reactive-mongo").',
+      solution: 'Check the library documentation, perhaps an option is missing.'
     };
   }
   return true;
 };
 
-OplogCheck.gitCheckout = function (cursorDescription, driver) {
-  if (!Meteor.release) {
+OplogCheck.unknownReason = function (cursorDescription, driver) {
+  if (driver && driver.constructor.name !== 'OplogObserveDriver') {
     return {
-      code: 'GIT_CHECKOUT',
-      reason: "Seems like your Meteor version is based on a Git checkout and it doesn't have the oplog support.",
-      solution: 'Try to upgrade your Meteor version.'
+      code: 'UNKNOWN_REASON',
+      reason: 'Not using the Oplog Observe Driver for unknown reason.',
+      solution: 'Check your third-party libraries.'
```

Comment:
Do you know of any third party libraries that can cause this?

Reply:
Not specifically, only that one would be the initial suspect.